src/offline.cpp

```diff
@@ -184,4 +184,18 @@ void OfflinePrivate::updateProperties(const QVariantMap &properties)
     }
 }
 
+void OfflinePrivate::updateProperties(const QString &interface, const QVariantMap &properties, const QStringList &invalidate)
+{
+    if(interface != PK_OFFLINE_INTERFACE) {
+        qCWarning(PACKAGEKITQT_OFFLINE) << "Cannot process" << interface << "as" << PK_OFFLINE_INTERFACE;
+        return;
+    }
+
+    if (!invalidate.isEmpty()) {
```

Comment:
shouldn't this clear the property value?

Reply:
Not really, it should re-request it if we ever need it.

But we are not using invalidates, so it feels unnecessary to add such logic.